components/widgets/settings/setting.tsx

```diff
@@ -34,12 +34,12 @@ const Setting: React.FC<Props> = ({
             >
                 {label}
             </label>
-            <div
-                data-testid={inputId + 'help-text'}
-                className={inputClassName}
-            >
+            <div className={inputClassName}>
```

Comment:
made a couple of changes during the merge - this being one of them the current master had the `test-id` on the parent element and then was doing a `find('span')` to get the actual help text I moved the test id to the child so we dont have to do the `find` anymore

Reply:
Thanks, works for me.